src/core/lib/iomgr/ev_epollex_linux.cc

```diff
@@ -510,6 +529,18 @@ static grpc_error* pollable_add_fd(pollable* p, grpc_fd* fd) {
   static const char* err_desc = "pollable_add_fd";
   const int epfd = p->epfd;
 
+  for (int i = 0; i < MAX_PROBE_EPOLL_FDS; i++) {
```

Comment:
Add some stats for cache-hits vs cache misses

Reply:
Added